.github/workflows/php-cs-fixer.yml

```diff
@@ -0,0 +1,25 @@
+name: PHP-CS-Fixer
+
+on:
+    pull_request:
+    push:
+
+jobs:
+    php-cs-fixer:
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.3
```

Comment:
let's run on lowest supported PHP for PHP Coveralls

Reply:
I think this have no reason because here we just need to run globally installed `php-cs-fixer`. Here we don't have any relation with project requirements

Reply:
as creator of PHP CS Fixer - run the PHP CS Fixer on the PHP that your project must work.
If you run PHP CS Fixer on PHP v8 and claim that your project works on PHP v5, it may be that PHP CS Fixer will apply the changes into codebase that will be valid for PHP v8, but not for PHP v5, eg `: void`.

Reply:
Ok, I will change

Reply:
@keradus FYI - only 5.6 allowed
```
 php-cs-fixer fix --dry-run --diff -v
  shell: /usr/bin/bash -e {0}
PHP needs to be a minimum version of PHP 5.6.0 and maximum version of PHP 7.3.*.
Error: Process completed with exit code 1.
```

Reply:
then let's run on 5.6.

we may want to run it on 8.x, but then we would need to be very careful about applied rules.